src/server/mod.rs

```diff
@@ -48,6 +50,8 @@ macro_rules! build_app {
             .wrap(Cors::permissive())
             // Next, the API we are implementing
             .service(web::resource("/v1/tiles").route(web::get().to(handlers::get_tiles)))
+            // image cache tester...
```

Comment:
Do we want this on or commented out? Or is there a `cfg(debug)` we can wrap it in?

Reply:
I'll comment it out for now. Again, mostly demo code.